Bai8.js

```diff
@@ -0,0 +1,25 @@
+let initArray = [
+    {name: "Bui", age: 25},
+    {name: "Viet", age: 26},
+    {name: "Hoang", age: 27},
+    {name: "Bui2"},
+    {age: 20},
+    {name: "Hoang2", age: 18}
+]
+
+function getAverageAge(arr) {
+    let ageArr = arr.map((item) => {
+        return item.age;
+    })
+    let total = 0;
+    ageArr.forEach((item) => {
+        if (Number.isInteger(item)) {
+            total += item;
+        }
+    })
```

Comment:
viết lại hàm này đi e, dùng tới 2 vòng for để xử lý 2 điều kiện, có thể 1 vòng xử lý được luôn mà e